### PR TITLE
Tests and further Python 3 support

### PR DESCRIPTION
**Pull request by kmu**
- README is updated for a better look at GitHub.
- Tests are added.
- A build script for Travis is added for your reference.
- Several bugs are fixed.

### CODE REVIEW

**Comment by ltalirz** [comment]
Thanks a lot @kmu !

I'll get back to this PR once @torbjornbjorkman has had the time to check out #4 .

One question that comes to mind is whether all testing should be moved to `pytest` - given the nice report functionalities of pytest, I think that would be a good idea.

I suggest to use pytest's [`parametrize`](https://docs.pytest.org/en/latest/parametrize.html) to loop over all the CIF files in the subdirectories.

In a second step (another PR), we should probably move everything except the command line parser out of the `cif2cell` script and into the `cif2cell` package (@torbjornbjorkman is already on board with this).

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    - 3.7
+    - 2.7
+
+install:
+  - pip install -e .
+
+script:
+  - cif2cell --help
+  - .ci/test_script.sh
+  - pytest tests
```
Comment (ltalirz) [inline .travis.yml:14]: Does this pass on travis?
Probably need to add a `tests` extra to the `setup.py` in order to get `pytest` and `pip install -e .[tests]`.

**Comment by torbjornbjorkman** [comment]
Great stuff!

**Comment by kmu** [comment]
Thanks, @torbjornbjorkman, @ltalirz!

**Pull request merged**